examples/dt/PIDONE/DFG-pidone-18.0-3cont-2comp-ipv6-geneve-HA/DFG-pidone-18.0-3cont-2comp-ipv6-geneve-HA.md

```diff
@@ -0,0 +1,43 @@
+RHOS Deployed Topology DFG-pidone-18.0-3cont-2comp-ipv6-geneve-HA
+Based on OpenStack K8S operators from the "main" branch of the OpenStack Operator repo on Oct 17th, 2023
+
+General information:
+Revision	Change		Publication Date
+v0.1	 	Initial		Dec 20 2023
+
+Node topology:
+Node role	                                bm/vm	amount
+Openshift master/worker combo-node cluster	vm	    3
+Compute node					vm	    2
+
+Services, enabled features and configurations:
+Service	        configuration	    Lock-in coverage?
+Galera      default             Must have
+RabbitMQ	default	            Must have
+Redis           ?                   Must have
+Memcached       default             Must have
+TLS             default             Must have
+FIPS Mode	default	            Must have/standard
+
+Considerations:
+Highly Available deployment
+Network protocol - ipv6
+
+Testing tree:
+Test framework			Stage to run	Special configuration	Test case to report
+Tobiko/Sanity (health checks)	stage7	        Use cirros image        33445566
+Tobiko/Resources' creation	stage7	        Use cirros image	33445566
+Tobiko/Faults	            	stage9	        Use cirros image	33445566
```

Comment:
In what state is the OSP cluster left after destructive tests?

Reply:
The cluster is expected to recover and all the resources/services running.

Reply:
After the faults tests, we would run 'Check resources' and 'Sanity' tests to check the healthiness of the OSP cluster and the workloads.

Reply:
Since faults tests may affect the environment, the Tobiko stage would rather to be placed at the end of the pipeline, after the Tempest stage.

Reply:
A common Tobiko stage flow:
- Sanity
- Create resources
- Faults
- Sanity after
- Check resources